training/nvidia-bootc/Makefile

```
@@ -30,4 +30,4 @@ bootc: driver-toolkit check-sshkey prepare-files
 		--security-opt label=disable \
 		--tag "${BOOTC_IMAGE}" \
 		-v ${OUTDIR}:/run/.input:ro \
-		${CONTAINER_TOOL_EXTRA_ARGS} .
+		${CONTAINER_TOOL_EXTRA_ARGS} ..
```

Does this work if called from the makefile in the parent directory?

---

like `make ./nvidia-bootc/Makefile bootc` ? If so it will fail early because of the Makefile inclusion

---

More of `make nvidia` in the training directory.